source/api/email.md

```diff
@@ -69,3 +93,30 @@ if you want to intercept emails sent by core packages like accounts-password
 or other packages where you can't modify the email code.
 
 The hook function will receive an object with the options for Nodemailer.
+
+{% apibox "Email.customTransport" %}
+
+> `Email.customTransport` is only available since Email v2.2
+
+There are scenarios when you have your own transport set up, be it an SDK 
+for your mailing service or something else. This is where `customTransport` 
+comes in. If you set this function all sending events will be passed to it
+(after `hookSend` is run) with an object of the options passed into `send` 
+function with addition of `settings` key which will pass in package settings
```

Comment:
`settings` here I believe should be `packageSettings` as well, right?

Reply:
fixed